sphinx/ext/imgmath.py

```diff
@@ -85,7 +85,7 @@ class InvokeError(SphinxError):
 '''
 
 depth_re = re.compile(br'\[\d+ depth=(-?\d+)\]')
-
+depthsvg_re = re.compile(br'.*, depth=(.*)pt')
```

Comment:
Here is some example output from dvisvgm when the dvi was produced with tightpage option of preview.

```
$ dvisvgm --no-fonts testsvg
pre-processing DVI file (format version 2)
processing page 1
  computing extents based on data set by preview package (version 11.91)
  width=103.354867pt, height=9.225002pt, depth=3.075001pt
  graphic size: 103.354867pt x 12.300003pt (36.325081mm x 4.322957mm)
  output written to testsvg.svg
1 of 1 page converted in 0.253939 seconds
```

Perhaps the regex can be made better?